roles/preflight/tasks/test_preflight_check_container_one_image.yml

```diff
@@ -77,6 +77,8 @@
             {% if cert_project_id | default('') | length %}
             --submit
             {% endif %}
+          retries: 2
+          delay: 120
```

Comment:
Needs an `until` condition in here otherwise the retry won't work [1]

[1] https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_loops.html#retrying-a-task-until-a-condition-is-met

Reply:
Added